pkg/products/marin3r/apiUsagePrometheusRules.go

```diff
@@ -157,8 +157,9 @@ func getRateLimitInSeconds(rateLimitUnit string, rateLimitRequestsPerUnit uint32
 	} else if rateLimitUnit == "day" {
 		return float64(rateLimitRequestsPerUnit) / (60 * 60 * 24), nil
 	} else {
-		logrus.Errorf("Unexpected Rate Limit Unit %v, while creating 3scale api usage alerts", rateLimitUnit)
-		return 0, errors.New(fmt.Sprintf("Unexpected Rate Limit Unit %v, while creating 3scale api usage alerts", rateLimitUnit))
+		err := errors.New(fmt.Sprintf("Unexpected Rate Limit Unit %v, while creating 3scale api usage alerts", rateLimitUnit))
+		r.log.Error("", err)
```

Comment:
Is the empty string, err intended? I see that the error string got created @160 but the empty string is a bit confusing.

Reply:
In most cases it's useful to pass context to the error log. In this case because we're creating the error there's no need. Anything here would just be superfluous.

Reply:
Makes sense, thank you.